qemu/tests/cfg/rebase_second_snapshot_to_base.cfg

```diff
@@ -0,0 +1,51 @@
+- rebase_second_snapshot_to_base:
+    only raw
+    virt_test_type = qemu
+    type = rebase_second_snapshot_to_base
+    kill_vm = yes
+    start_vm = no
+    force_create_image = no
+    # md5sum binary path
+    md5sum_bin = "md5sum"
+    image_chain = "image1 sn1 sn2"
+    image_format_image1 = "raw"
+    image_name_sn1 = "images/sn1"
+    image_format_sn1 = qcow2
+    backing_chain_sn1 = yes
+    image_name_sn2 = "images/sn2"
+    image_format_sn2 = qcow2
+    backing_chain_sn2 = yes
+    # set size to "", so during snapshot creation
+    # the cmdline will not have specified size option
+    image_size_sn1 = ""
+    image_size_sn2 = ""
+    tmp_dir = /var/tmp
+    tmp_file_name = ${tmp_dir}/testfile
+    file_create_cmd = "dd if=/dev/urandom of=%s bs=1M count=512"
+    guest_file_name = ${tmp_file_name}
+    Windows:
+        guest_file_name = C:\testfile
+        x86_64:
+            sync_bin = WIN_UTILS:\Sync\sync64.exe /accepteula
+        i386, i686:
+            sync_bin = WIN_UTILS:\Sync\sync.exe /accepteula
+    variants:
+        - cache_mode:
+            variants:
+                - writethrough:
+                    cache_mode = writethrough
+                - writeback:
+                    cache_mode = writeback
+                - none:
+                    cache_mode = none
+                - unsafe:
+                    cache_mode = unsafe
+                - directsync:
+                    cache_mode = directsync
+        - compat_level:
```

Comment:
It is better to filter out the qemu which doesn't support this option. For those qemu, we can simply skip the tests.

Reply:
@hereischen @hellenmao qcow2v3 was supported since qemu-1.1.
Detail information could be checked from upstream changelog as below
https://wiki.qemu.org/ChangeLog/1.1

Reply:
required_qemu = [1.1, )